aws-android-sdk-appsync-api/src/main/java/com/apollographql/apollo/api/InputType.java

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
Can you isolate your fixes and undo the cache key changes from https://github.com/awslabs/aws-mobile-appsync-sdk-android/pull/146/commits/15840b75a837acdf42c0c0fe045811f9571fbae5. There is separate PR with these changes which is still a work in progress.

Reply:
I've reverted the changes from 15840b75a837acdf42c0c0fe045811f9571fbae5